routes/api.js

```diff
@@ -233,8 +233,9 @@ router.post('/leaderBoard', authenticationHelpers.isAuthOrRedirect, (req, res) =
             .then(data => {
                 let array = [];
                 data.forEach(function(item) {
-                    array.push({name: `${item.user_account.fname} ${item.user_account.lname}`,
-                        score: item.score, userEmail: item.user_account.email})
+                    let name;
+                    name = (item.user_account.lname) ? `${item.user_account.fname  } ${  item.user_account.lname}` : item.user_account.fname;
```

Comment:
Remove the extra spaces after `item.user_account.fname` and before `item.user_account.lname`

Reply:
@paavininanda done the changes.